modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -180,14 +180,16 @@ https://eu1.anypoint.mulesoft.com/exchange/api/v1/assets/ORGANIZATION_ID/ASSET_I
 
 == Publish Federated Assets
 
-. Get the access token by logging into Anypoint Platform and copying this URL to the browser address bar:
+. xref:access-management::saml-bearer-token.adoc[Get an API bearer token by using a SAML assertion].
```

Comment:
Should this be in title case?

QUESTION: When we use xrefs in sentences, we alter them to fit into the sentence. Does that affect the actual title?

Reply:
The title of that page is "To Obtain an API Bearer Token Using a SAML Assertion", and I rewrote it to fit this text, including using regular non-title case. The xref does not affect the actual title.